### DISCUSSION TITLE
Show dashboard with live data from API

### DISCUSSION THREAD

**Discussion by mirzaehtishamdanish** [lowdefy/lowdefy#1015] [Questions and Help]
Hi,

I want to show real time data like weather results update on graph, how to notify my graph to get update data from rest API? could you please gives a instruction how to achieve this task.

**Comment by Gervwyk** [comment]
Hi @mirzaehtishamdanish 

You can see this discussion https://github.com/lowdefy/lowdefy/discussions/878 
We have hacked together a solution here for data polling here, but might introduce more official config for this in the future.
Let me know if you have any questions.

**Reply by mirzaehtishamdanish** [reply]
Is there any custom option available to render graph with data polling
